config/app.default.php

```diff
@@ -9,7 +9,7 @@
      * Development Mode:
      * true: Errors and warnings shown.
      */
-    'debug' => true,
+    'debug' => env('DEBUG', true),
```

Comment:
Good point. I prefer prefixing if the variable is too generic. I can think of many scenarios for the conflicts for the generic `DEBUG`.